src/i18ndude/untranslated.py

```diff
@@ -201,6 +202,15 @@ def endElement(self, tag):
                         # including literal content, that does not need to be
                         # translated.
                         pass
+                    obj = re.match('\${.*}', data)
+                    if obj:
+                        stripped = data.replace(
+                            obj.group(0), '').replace(' ', '')
```

Comment:
Should maybe iterate through in case there are multiple instances of the syntax in the string data..?